compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.td

```diff
@@ -226,7 +226,7 @@ def IREECodegen_TranslationInfoAttr :
   }];
 
   let assemblyFormat = [{
-    `<` `pipeline` `=` $passPipeline
+    `<` `pipeline` `=` `` $passPipeline
```

Comment:
Should this be here?

Reply:
it is - it affects printer/parser and thus affects attribute binding (it's due to a [bug upstream](https://github.com/llvm/llvm-project/pull/89249#issuecomment-2419849158) i/we/no one has fixed and probably never will)

Reply:
fair enough